src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -28,4 +28,16 @@ public function testDoNoDuplicateDefaultFormResources()
 
         $this->assertEquals(array('form_div_layout.html.twig'), $config['form_themes']);
     }
+
+    public function testArrayKeysInGlobalsAreNotNormalized()
+    {
+        $input = array(
+            'globals' => array('some-global' => array('some-key' => 'some-value')),
```

Comment:
Can you avoid using a dash in the global name itself, to make the test less confusing ?

Reply:
@stof I consciously did that to ensure that the normalization is also not happening for the global names themselves

Reply:
I would split that into two tests and use a meaningful test name then to make the intention more clear.

Reply:
Split the test